src/assets/css/sections/benefits.css

```diff
@@ -0,0 +1,19 @@
+.benefits-section {
+    position: relative;
+    top: -218px;
```

Comment:
don't hard code like here, please use another way

Reply:
can you suggest me ?